app.js

```diff
@@ -34,24 +34,22 @@ app.delete("/api/comments/:comment_id", removeCommentById);
 app.get("/api/users", getUsers);
 
 app.use((err, req, res, next) => {
-  if (err.msg === "Not Found") {
-    res.status(404).send({ msg: err.msg });
-  } else if (err.code === "22P02") {
+  if (err.code === "22P02") {
     res.status(400).send({ msg: "Bad Request" });
   }
-  if (err.msg === "article not found") {
+  if (
+    err.msg === "article not found" ||
```

Comment:
We could probably just say if (err.msg) send that message, as any message is being accepted here right?
If you want the status code to vary, we can res.status(err.status).send({msg: err.msg})